### ISSUE TITLE
The events pagination links fail colour contrast

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#87]
http://talarts.knowbility.org/643/

@christosptr, @ruchitamanuja  : Can you please suggest a colour?

![image](https://github-cloud.s3.amazonaws.com/assets%2F15111528%2F10957780%2Fb0aa1faa-83bb-11e5-8e84-ca685da0e765.png)

**Comment by mr-bobz** [comment]
Fixed

**Comment by HerinHentry** [comment]
Works. Issue can be closed.
![image](https://cloud.githubusercontent.com/assets/15111528/11015171/9b88840c-85a7-11e5-8666-b5a0e4ae5907.png)
